rulesdoc.md

```diff
@@ -378,45 +396,49 @@ Finally, choose 2 of the following bonuses.
 - Learn 1 Feat
 - Learn a maneuver and gain 2 proficiency with a weapon type
 - Increase your max wounds by 1
+- Increase your spell power by 5 and gain 3 mana
 
 ##### Variant Rule - Rolling for Attributes
 Instead of using the rules above for attributes, your group may decide to roll your scores. For each attribute roll 1 Superior, 3 Normal, and 1 Terrible die, adding up the total successes. The minimum score for an attribute is -2, with any rolls below that counting as -2 for the roll.
 
 ### Starting Equipment.
-Depending on your background you may start with additional gear. All players start with an adventuring kit containing a bedroll, 10 days of rations, a fire starter and 50ft of rope. In addition you start with 50 gold which you can spend on gear or upgrades. Any unspent gold becomes your characters starting money.
+Depending on your background you may start with additional gear. All players start with an adventuring kit containing a bedroll, 10 days of rations, a fire starter and 50ft of rope. In addition you start with 50 silver which you can spend on gear or upgrades. Any unspent silver becomes your characters starting money.
+
+#### Currency
+Currency in Farhome is divided into 3 types: the copper trite, silver cross, and platinum talent. An iron cross is worth 100 copper trites, and a platinum talent is worth 100 silver crosses.
 
 | Cost | Upgrade |
 |:----:|:-------------|
-| 5g  | Buy any simple weapon. |
-| 10g | Cantrip or Level 1 Spell |
-| 10g | Basic Healing Potion |
-| 10g | Buckler |
-| 15g | Padded or Studded armor. |
-| 15g | any martial weapon. |
-| 20g | Upgrading armor to the next type |
+| 5sc  | Buy any simple weapon. |
+| 10sc | Cantrip or Level 1 Spell |
+| 10sc | Basic Healing Potion |
+| 10sc | Buckler |
+| 15sc | Padded or Studded armor. |
+| 15sc | any martial weapon. |
+| 20sc | Upgrading armor to the next type |
 
 ### Leveling Up
 
-When you earn 1000 experience your character levels up. Each level you gain 2 proficiency points and choose one of the following bonuses:
+When you earn 1000 experience your character levels up. Each level you gain 1 proficiency point and choose one of the following bonuses:
```

Comment:
With the new dice rolling it isn't as punishing to not have proficiency. In our game it seems like as soon as a skill or ability becomes useful its proficiency is almost instantly maxed out. This doesn't feel good progression wise and players probably cap out around 15th level for skills they care about.

with this, not only is progression more gradual, it also makes taking proficiency points as a level up perk more valuable as opposed to the complete trash it is now. This should also help with removing skill overlap between characters as not everyone can be good at everything.

**This won't effect your current characters**. This is more of a balance change for next time.